src/seedu/addressbook/AddressBook.java

```diff
@@ -199,7 +199,10 @@
      * ====================================================================
      */
     public static void main(String[] args) {
```

Comment:
Remember to test the smelly code. A smelly code should still be a working code. Many more things can be undone to make codebase smellier.. comments, variable names, method names, abstracting messages by variable names, etc. think about what became difficult now after the making the code smelly.